README.md

```diff
@@ -1,134 +1 @@
-
-### Databases
-| Problems | Level | Link |
-|-|-|-|
-| [Trips and Users](/LeetcodeProblems/Databases/Trips_and_Users.sql) | Hard | https://leetcode.com/problems/trips-and-users/ |
-| [Human Traffic of Stadium](/LeetcodeProblems/Databases/Human_Traffic_of_Stadium.sql) | Hard | https://leetcode.com/problems/human-traffic-of-stadium |
-| [Rank Scores](/LeetcodeProblems/Databases/Rank_Scores.sql) | Medium | https://leetcode.com/problems/rank-scores |
-| [Consecutive Numbers](/LeetcodeProblems/Databases/Consecutive_Numbers.sql) | Medium | https://leetcode.com/problems/consecutive-numbers |
-| [Department Highest Salary](/LeetcodeProblems/Databases/Department_Highest_Salary.sql) | Medium | https://leetcode.com/problems/department-highest-salary |
-| [Exchange Seats](/LeetcodeProblems/Databases/Exchange_Seats.sql) | Medium | https://leetcode.com/problems/exchange-seats |
-| [Nth Highest Salary](/LeetcodeProblems/Databases/nth_Highest_Salary.sql) | Medium | https://leetcode.com/problems/nth-highest-salary |
-| [Combine Two Tables](/LeetcodeProblems/Databases/Combine_Two_Tables.sql) | Easy   | https://leetcode.com/problems/combine-two-tables |
-| [Second Highest Salary](/LeetcodeProblems/Databases/Second_highest_salary.sql)| Easy   | https://leetcode.com/problems/second-highest-salary |
-| [Customers Who Never Order](/LeetcodeProblems/Databases/Customers_Who_Never_Order.sql)| Easy |  https://leetcode.com/problems/customers-who-never-order |
-| [Reformat Department Table](/LeetcodeProblems/Databases/Reformat_Department_Table.sql) | Easy | https://leetcode.com/problems/reformat-department-table |
-| [Employees Earning More Than Their Managers](/LeetcodeProblems/Databases/Employees_Earning_More_Than_Their_Managers.sql) | Easy | https://leetcode.com/problems/employees-earning-more-than-their-managers/ |
-| [Delete Duplicate Emails](LeetcodeProblems/Databases/Delete_Duplicate_Emails.sql) | Easy | https://leetcode.com/problems/delete-duplicate-emails |
-| [Rising Temperature](LeetcodeProblems/Databases/Rising_Temperature.sql) | Easy | https://leetcode.com/problems/rising-temperature |
-
-### UtilsClasses
-
-Other languages provides built-in classes (e.g Linked List, Tree, etc). This module contains util classes to use in your problems. 
-
-### Contributions
-
-I'd be pleased to accept contributions. I'd be happy to discuss problems and solutions over a Pull Request or an Issue.
-
-Each problem should have:
-
-1) A description of the problem at the top of the file.
-2) A test file with some test cases. The test file must export a `test()` function which should run all the tests of the file.
-3) An entry of the problem in the list of solutions within the README file.
-
-PR Example: https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript/pull/39
+ [Maximal Square ](/LeetcodeProblems/Maximal_Square.js)                                               | Medium | https://leetcode.com/problems/maximal-square/ 
```

Comment:
Could you update the readme? It deleted all the previous staff.

Reply:
Yes did it now!